### ISSUE TITLE
[v4] FileView: menu glitches after resizing window (Safari)

### ISSUE THREAD

**Issue by manuelmoreale** [k-next/kirby#5606]
## Description
If the window is resized horizontally and then sized up again the menu seems to reappear with an extra dark overlay. From what I can see this is only happening in the file view and not in other parts of the panel.

**Screenshots**  
https://github.com/getkirby/kirby/assets/1738077/9cbc1b37-76a4-435b-a3e4-a44da7c2c554

## Your setup
Kirby 4 Beta 1

**Your system (please complete the following information)**
- OS: macOS 13.5.1
- Browser: Safari 16.6

**Comment by distantnative** [comment]
@manuelmoreale I can't reproduce this one. Anything in the console when that happens? Also, anything you did to the menu, e.g. as the bottom entries are missing?

**Comment by manuelmoreale** [comment]
@distantnative did some digging and apparently simply adding this field to the default file blueprint triggers the bug in my installation. Without it the bug's gone.

```
fields:
  alt:
    label : Alt Text
    type  : text
```

**Comment by manuelmoreale** [comment]
And just to be extra sure, I can reproduce this on a fresh starterkit. Weirdly enough if you add a second text field the bug is still there. It's also still there if you add a URL field but it goes away when a textarea is present.

https://github.com/getkirby/kirby/assets/1738077/28ef7125-938b-405d-bac5-adcb8b3dd3f8

**Comment by manuelmoreale** [comment]
Forgot to answer your question. I did absolutely nothing to the menu. I have a custom kit and I simply replaced the Kirby 3 folder with a Kirby 4 Beta 1 and deleted caches, media and all that. The installation is almost empty as I literally just started working on this site.

I'm gonna add the the icons at the bottom were probably just cropped out of the screen rercording but are actually there.
